src/main/java/aisu/parser/Parser.java

```diff
@@ -0,0 +1,56 @@
+package aisu.parser;
+
+import aisu.tasklist.TaskList;
+import aisu.command.AddCommand;
+import aisu.command.Command;
+import aisu.command.DeleteCommand;
+import aisu.command.ExitCommand;
+import aisu.command.FindCommand;
+import aisu.command.MarkDoneCommand;
+import aisu.command.ShowListCommand;
+import aisu.exception.AisuException;
```

Comment:
Lexicographical ordering would be good as before
```suggestion
import aisu.command.AddCommand;
import aisu.command.Command;
import aisu.command.DeleteCommand;
import aisu.command.ExitCommand;
import aisu.command.FindCommand;
import aisu.command.MarkDoneCommand;
import aisu.command.ShowListCommand;
import aisu.exception.AisuException;
import aisu.tasklist.TaskList;
```